js/src/api/util/address.js

```diff
@@ -0,0 +1 @@
+../../abi/util/address.js
```

Comment:
Not sure I fully followed the discussion about what goes in ABI vs. API. However it just looks weird to have a symlink to symlink.